commands/cookiecount/index.js

```diff
@@ -91,7 +91,7 @@ module.exports = {
 			}
 	
 			return {
-				reply: `${who} eaten ${total} cookies so far. Out of those, ${cookies.Received} were gifted to ${target}${voidString}. ${giftedString} ${reaction}`
+				reply: `${who} eaten ${total} cookies so far. Out of those, ${cookies.Received} were gifted to ${target}${voidString}. ${giftedString} ${reaction} ❗ This command will soon be archived. Use => $stats cc`
```

Comment:
We probably want to include full usage of the command in the deprecation notice, so `$stats cc/tcc (user)` based on the current invocation, with the user included.